packages/bpk-component-datatable/src/bpkHeaderRenderer.js

```diff
@@ -35,23 +37,35 @@ const downIconClassName = getClassName(
   'bpk-data-table-column__sort-icon--down',
 );
 
-export const getSortIconDirection = element => {
+export const getSortIconDirection = (element: Element): ?SortDirectionType => {
   if (
     hasClassName(element, upIconClassName) ||
-    hasClassName(element.parentNode, upIconClassName)
+    hasClassName(element.parentElement, upIconClassName)
```

Comment:
Is this change due to the type that we are assigning `element`?

Reply:
Yes, The type of `parentNode` is `Node` whereas the type of `parentElement` is `Element` we need it to be element because that's where we get `getAttributes` from which is used by `hasClassName`. `parentNode` and `parentElement` is the exact same instance here so this is fine